webapp/channels/src/packages/mattermost-redux/src/actions/channel_bookmarks.ts

```diff
@@ -83,6 +83,26 @@ export function editBookmark(channelId: string, id: string, patch: ChannelBookma
     };
 }
 
+export function reorderBookmark(channelId: string, id: string, newOrder: number, connectionId: string) {
```

Comment:
Could this use `bindClientFunc`, or is that not possible because we need to pass the `channelId` as part of the `RECEIVED` action as well?